### PR TITLE
Version update 0.4.3

### PR DESCRIPTION
**Pull request by imransl**
Updated package to ~`0.4.2`~ `0.4.3`. The related ensembl-client PR is: https://github.com/Ensembl/ensembl-client/pull/782

JIRA: [ENSWBSITES-1665](https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-1665)

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -357,6 +357,7 @@ export type GenomeBrowserType = {
   goto: (left: number, right: number) => void;
   jump: (location: string) => void;
   set_y: (y: number) => void;
+  switch: (path: string[], value: string[] | boolean | null) => void;
```
Comment (azangru) [inline src/types.ts]: The type of `value` in `switch` is any valid JSON value, see https://github.com/Ensembl/ensembl-dauphin-style-compiler/blob/3460dc08ded403007a9b026700c231f16276b791/peregrine-toolkit/src/js/jstojsonvalue.rs#L30-L41

File: src/types.ts [hunk 2]
```diff
@@ -357,6 +357,7 @@ export type GenomeBrowserType = {
...
   set_switch: (path: string[]) => void;
```
Comment (azangru) [inline src/types.ts]: Should this be removed?

Comment (imransl) [reply src/types.ts]: Yes think we don't need both `set_switch` and `clear_switch` anymore

File: src/types.ts [hunk 3]
```diff
@@ -350,13 +351,22 @@ export type Subscribe = (...args: SubscribeArgs) => {
   unsubscribe: () => void
 };
 
+export type PrimitiveValue = string | number | boolean | null;
+export type ArrayValue = PrimitiveValue[] | JSONValue[];
+export type PrimitiveOrArrayValue = PrimitiveValue | ArrayValue;
+
+export type JSONValue = PrimitiveOrArrayValue | {
+  [key: string]: PrimitiveOrArrayValue | JSONValue;
+};
```
Comment (imransl) [inline src/types.ts:361]: Not sure if I've got this correct. `genomeBrowser.switch` is not happy in `send.ts` with the `JSONValue` type that is similar to the one in `ensembl-client`. So had to add in `PrimitiveOrArrayValue` as well.

Comment (azangru) [reply src/types.ts:361]: ~Peregrine is also capable of handling json objects... Don't know if Dan intends to; but his function parses objects as well.~

Sorry, ignore me. It looks correct.

**Review by msmanoj — APPROVED** [review]

**Review by azangru — APPROVED** [review]

File: package-lock.json [hunk 4]
```diff
@@ -1,12 +1,12 @@
 {
   "name": "@ensembl/ensembl-genome-browser",
-  "version": "0.4.1",
+  "version": "0.4.2",
```
Comment (ridwanamo) [inline package-lock.json]: this is still 0.4.2 ?

Comment (azangru) [reply package-lock.json]: 😱

Comment (azangru) [reply package-lock.json]: Updated.

**Review by ridwanamo — APPROVED** [review]

**Pull request merged**